python/paddle/v2/optimizer.py

```diff
@@ -64,8 +69,12 @@ def create_updater(self, is_local, num_passes, use_sparse_updater):
         if is_local:
             parameter_updater = self.__create_local_updater__()
         else:
-            parameter_updater = self.__create_remote_updater__(
-                num_passes, use_sparse_updater)
+            if pserver_spec is None:
+                parameter_updater = self.__create_remote_updater__(
+                    num_passes, use_sparse_updater)
+            else:
+                parameter_updater = self.__create_new_remote_updater__(
+                    pserver_spec)
```

Comment:
I see, the current code may be old here?
https://github.com/PaddlePaddle/Paddle/blob/develop/go/pserver/cclient/cclient.go#L103
```go
func paddle_new_pserver_client(addrs *C.char, selected int) C.client {
```